File: Modul 6/Python/PRAK605-2310817310005-MuhammadNurwahyudiAdhitama.py
import numpy as np

ordo = int(input("Masukkan jumlah ordo matriks : "))

print("Matriks A:")
E_ARRAY1 = []
for i in range(ordo):
    BARIS = list(map(int, input("").split()))
    E_ARRAY1.append(BARIS)

print("Matriks B:")
E_ARRAY2 = []
for i in range(ordo):
    BARIS = list(map(int, input("").split()))
    E_ARRAY2.append(BARIS)

arr1 = np.array(E_ARRAY1).reshape(ordo,ordo)
arr2 = np.array(E_ARRAY2).reshape(ordo,ordo)

print("Matriks AXB : ")

PERKALIAN = np.dot(arr1, arr2)

for HASIL in PERKALIAN :
    print(*HASIL) 

File: Modul 6/C/PRAK605-2310817310005-MuhammadNurwahyudiAdhitama.c
#include <stdio.h>

void main() {
    int ordo,baris1,baris2;

    scanf("%d", &ordo);

    int arrA[ordo][ordo];
    int arrB[ordo][ordo];

    printf("Matriks A : \n");
    for (int i = 0 ; i < ordo ; i++) {
        for (int j = 0 ; j < ordo ; j++) {
            scanf("%d", &arrA[i][j]);
        }
    }

    printf("Matriks B : \n");
    for (int i = 0 ; i < ordo ; i++) {
        for(int j = 0 ; j < ordo ; j++) {
            scanf("%d", &arrB[i][j]);
        }
    }

    int HASIL[ordo][ordo];

    for (int i = 0; i < ordo; i++) {
        for (int j = 0; j < ordo; j++) {
            HASIL[i][j] = 0;
            for (int h = 0; h < ordo; h++) {
                HASIL[i][j] += arrA[i][h] * arrB[h][j];
            }
        }
    }

    printf("Matriks AXB : \n");
    for (int i = 0; i < ordo; i++) {
        for (int j = 0; j < ordo; j++) {
            printf("%d ", HASIL[i][j]);
        }
        printf("\n");
    }
}